core/src/test/java/org/frankframework/jdbc/StoredProcedureQuerySenderTest.java

```diff
@@ -32,46 +30,85 @@
 import java.util.zip.DeflaterInputStream;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
 import org.frankframework.core.PipeLineSession;
 import org.frankframework.core.SenderException;
 import org.frankframework.core.SenderResult;
 import org.frankframework.dbms.JdbcException;
 import org.frankframework.parameters.Parameter;
 import org.frankframework.stream.Message;
 import org.frankframework.testutil.TestFileUtils;
+import org.frankframework.testutil.TransactionManagerType;
+import org.frankframework.testutil.junit.DatabaseTest;
+import org.frankframework.testutil.junit.DatabaseTestEnvironment;
+import org.frankframework.testutil.junit.WithLiquibase;
+import org.frankframework.util.LogUtil;
 import org.frankframework.util.StreamUtil;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 
-public class StoredProcedureQuerySenderTest extends JdbcTestBase {
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import liquibase.database.Database;
+import liquibase.database.DatabaseFactory;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.ClassLoaderResourceAccessor;
 
+@WithLiquibase(tableName = StoredProcedureQuerySenderTest.TABLE_NAME, file = "Jdbc/StoredProcedureQuerySender/DatabaseChangelog-StoredProcedures.xml")
+public class StoredProcedureQuerySenderTest {
+
+	protected static final String TABLE_NAME = "SP_TESTDATA";
 	public static final String TEST_DATA_STRING = "tést-data";
 	StoredProcedureQuerySender sender;
+	protected static Logger log = LogUtil.getLogger(StoredProcedureQuerySenderTest.class);
+	protected Liquibase liquibase;
 
 	PipeLineSession session;
 
-	@Before
-	public void setUp() throws Exception {
-		runMigrator("Jdbc/StoredProcedureQuerySender/DatabaseChangelog-StoredProcedures.xml");
+	@DatabaseTest.Parameter(0)
+	private TransactionManagerType transactionManagerType;
```

Comment:
These tests now run for every database, for every transaction-manager-type.
They only need to run for every database.

@nielsm5 If the transactionManagerType is removed, do they then run automatically just for each DB? Instead of the full matrix of every DB + every TX manager type?